applications/openshift/api-server/var_apiserver_encryption_path.var

```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
+
+title: 'OpenShift APIServer etcd encryption path'
```

Comment:
Can you give me an example of the variable value for hypershift? Currently the name seems odd to be because the default value points to the apiserver object instance, but the variable name and title say `etcd`

Reply:
etcd encryption is enabled in APIserver object: https://docs.openshift.com/container-platform/4.12/security/encrypting-etcd.html#enabling-etcd-encryption_encrypting-etcd

Where to check on HyperShift: `/apis/hypershift.openshift.io/v1beta1/namespaces/clusters/hostedclusters/<hypershift-hosted-cluster-name>`

Reply:
OK, I wasn't really explicit, what I'm wondering is if we should make the variable name more generic. Right now it's used only for rules that are related to etcd encryption, so it makes sense, but is it likely that the variable will be reused on other more generic rules?

Reply:
I think that is possible that other rule in the future might be able to use this var